src/OpenLoco/src/Ui/Windows/Vehicle.cpp

```diff
@@ -1135,55 +1169,96 @@ namespace OpenLoco::Ui::Windows::Vehicle
                     GameCommands::doCommand(gcArgs, GameCommands::Flags::apply);
                     break;
                 }
+                case widx::paintBrush:
+                    if (Input::hasKeyModifier(Input::KeyModifier::shift))
```

Comment:
i don't see a point to having shift+click on the brush to paint the entire train when you can just shift+click on the train itself

Reply:
It saves having to click on the train itself. I like shift-click as an alternate behavior modifier. The modify vehicle button IMO should have shift-click clone so you don't have to deal with the dropdown.

Reply:
i don't agree with it - having shift-click behaviour on everything is not intuitive to the player as they have no visual indicator that additional behaviour exists. hence why i suggested the tooltip for the colouring, because it makes sense to use key modifiers here, so we should inform the player.

as mentioned in the other PR, we should solve that with a dedicated button instead

Reply:
Pull the latest changes. The visual indicator is the dropdown.